src/controllers/things_controller.js

```diff
@@ -144,6 +144,52 @@ ThingsController.ws('/:thingId/', function(websocket, request) {
   websocket.on('close', function() {
     AdapterManager.removeListener('property-changed', onPropertyChanged);
   });
+
+  websocket.on('message', function(requestText) {
+    let request = null;
+    try {
+      request = JSON.parse(requestText);
+    } catch(e) {
+      websocket.send(JSON.stringify({
+        messageType: Constants.ERROR,
```

Comment:
Constants.ERROR doesn't actually seem to be set, did you forget to check in constants.js?